front-end/src/components/Card/Card.js

```diff
@@ -1,23 +1,90 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import moment from 'moment';
+import { WatchLater, ThumbUpAlt, ThumbUpAltOutlined } from '@material-ui/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData } from '../../redux/nodes/authentication/selectors';
+import { likePost, unlikePost } from '../../redux/nodes/entities/posts/actions';
+import { getComments } from '../../redux/nodes/entities/comments/selectors';
+import { getUsers } from '../../redux/nodes/entities/users/selectors';
 import { StyledCard, StyledCardContent } from './styled';
 import { propTypes, defaultProps } from './props';
+import { Comment } from './Comment';
+import { GridContainer, GridItem } from '../Grid';
 import { Text } from '../Text';
+import { Button } from '../Button';
+import { CreateCommentForm } from '../Forms';
+
+export const Card = ({ post, ...rest }) => {
+  const dispatch = useDispatch();
+  const { userId } = useSelector(getData);
+  const { comments } = useSelector(getComments);
+  const { users } = useSelector(getUsers);
+  const [liked, setLiked] = useState(false);
```

Comment:
state name should be likesCount

Reply:
Liked is actually a flag to check if the post has been liked. This state has been renamed to postLiked.